### PR TITLE
Translate strings to Chinese

### PR DESCRIPTION
**Pull request by vfishv**
translate strings to Chinese
androidGradlePlugin: '8.2.0'. bump to  androidGradlePlugin: '8.2.2'

### CODE REVIEW

File: platform/android/java/app/config.gradle
Comment (AThousandShips) [inline platform/android/java/app/config.gradle:1]: This should be a separate PR IMO, it's unrelated to translations

**Review by m4gr3d — CHANGES_REQUESTED** [review]
Can you address the feedback from @AThousandShips

File: platform/android/java/lib/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+
+    <string name="state_unknown">启动…</string>
+    <string name="state_idle">等待下载开始</string>
+	<string name="state_fetching_url">寻找要下载的资源</string>
+	<string name="state_connecting">连接到下载服务器</string>
+	<string name="state_downloading">下载资源</string>
+	<string name="state_completed">下载完成</string>
+	<string name="state_paused_network_unavailable">由于没有可用网络，下载已暂停</string>
+	<string name="state_paused_network_setup_failure">下载已暂停。 在浏览器中测试网站</string>
```
Comment (timothyqiu) [inline platform/android/java/lib/res/values-zh-rCN/strings.xml:42]:
```suggestion
	<string name="state_paused_network_setup_failure">下载已暂停。在浏览器中测试网站</string>
```

File: platform/android/java/lib/res/values-zh-rCN/strings.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	<string name="state_paused_by_request">下载已暂停</string>
+	<string name="state_paused_wifi_unavailable">由于 wifi 不可用，下载已暂停</string>
+	<string name="state_paused_wifi_disabled">下载已暂停，因为 wifi 已禁用</string>
+	<string name="state_paused_roaming">下载已暂停，因为您正在漫游</string>
```
Comment (timothyqiu) [inline platform/android/java/lib/res/values-zh-rCN/strings.xml:46]:
```suggestion
	<string name="state_paused_roaming">下载已暂停，因为你正在漫游</string>
```

File: platform/android/java/lib/res/values-zh-rCN/strings.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="text_paused_cellular">您想启用通过蜂窝连接下载吗？ 根据您的数据计划，这可能会花费您金钱。</string>
+    <string name="text_paused_cellular_2">如果您选择不启用通过蜂窝连接下载，则当 Wi-Fi 可用时，下载将自动恢复。</string>
```
Comment (timothyqiu) [inline platform/android/java/lib/res/values-zh-rCN/strings.xml:4]: For overall consistency with other translated parts of Godot, please use `你` instead of `您` for the second personal pronoun.

Also applies to other strings.

File: platform/android/java/lib/res/values-zh-rCN/strings.xml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    <string name="text_button_wifi_settings">Wi-Fi 设置</string>
+    <string name="text_verifying_download">验证下载</string>
+    <string name="text_validation_complete">XAPK 文件验证完成。 选择确定退出。</string>
+    <string name="text_validation_failed">XAPK 文件验证失败。</string>
+    <string name="text_button_pause">暂停下载</string>
+    <string name="text_button_resume">继续下载</string>
+    <string name="text_button_cancel">取消</string>
+    <string name="text_button_cancel_verify">取消验证</string>
+    <string name="text_error_title">错误!</string>
+    <string name="error_engine_setup_message">无法启动 Godot 引擎！ 正在退出…</string>
```
Comment (timothyqiu) [inline platform/android/java/lib/res/values-zh-rCN/strings.xml:15]:
```suggestion
    <string name="error_engine_setup_message">无法启动 Godot 引擎！正在退出…</string>
```

**Review by timothyqiu — CHANGES_REQUESTED** [review]
Some issues about the translation.
